### ISSUE TITLE
sysrsync breaks when a path contains a whitespace symbol

### ISSUE THREAD

**Issue by philip-bl** [gchamon/sysrsync#22]
If I invoke sysrsync like `sysrsync.run(source="/mnt/foo/bar bar/buzz", destination="/mnt/dest")`, it doesn't pass the source path `/mnt/foo/bar bar/buzz` as a single argument to rsync. Instead rsync receives it as two arguments `/mnt/foo/bar` and `/bar/buzz` and thus it exits with an error.

The version of sysrsync used is 1.0.1.

**Comment by philip-bl** [comment]
I think this can be fixed by just modifying runner.py in the following way

```
import os
import subprocess

from sysrsync.command_maker import get_rsync_command
from sysrsync.exceptions import RsyncError


def run(cwd=os.getcwd(), strict=True, verbose=False, **kwargs):
    rsync_command = get_rsync_command(**kwargs)
    # rsync_string = ' '.join(rsync_command)

    if verbose is True:
        print(f'[sysrsync runner] running command on "{cwd}":')
        # print(rsync_string)
        print(rsync_command)
    # process = subprocess.run(rsync_string, cwd=cwd, shell=True)
    process = subprocess.run(rsync_command, cwd=cwd, shell=False)

    if strict is True:
        code = process.returncode
        _check_return_code(code, rsync_command)

    return process


# def _check_return_code(return_code: int, action: str):
def _check_return_code(return_code: int, action: List[str]):
    if return_code != 0:
        raise RsyncError(f"[sysrsync runner] {action} exited with code {return_code}")
```

**Comment by gchamon** [comment]
I added `shell=True` to deal with rsh
I have to revisit this decision to remove the use of shell

This also has security implications, so removing shell is the most adivisable course of action, but I don't want to introduce regressions.

I would also like to write some end-to-end tests before removing shell.

**Comment by gchamon** [comment]
fixed with 1.1.0 release
